demo/api/src/app.module.ts

```diff
@@ -79,7 +79,7 @@ export class AppModule {
                 UserPermissionsModule.forRootAsync({
                     useFactory: (userService: UserService, accessControlService: AccessControlService) => ({
                         availableContentScopes: [
-                            { domain: "main", language: "de" },
+                            { domain: "main", language: "de1" },
```

Comment:
that was just for testing i suppose?
```suggestion
                            { domain: "main", language: "de" },
```